etc/rc.conf

```diff
@@ -0,0 +1,17 @@
+###############################################################################
+# rc.conf is loaded by /lib/init/rc.boot and /lib/init/rc.shutdown as early
+# as possible. Please inspect these scripts for more details.
+#
+# Essentially, rc.conf is for config variables, and /etc/rc.d/ is for scripts.
+# Except for shutdown hooks, rc.d/ scripts should be suffixed with .sh
+# rc.d/ scripts when executable are always exececuted. Please use caution.
+#
+# Example 1: The following creates a simple dmesg save boot script,
+# -> echo "dmesg > /var/log/dmesg.log" > /etc/rc.d/dmesg.sh
+#
+# Example 2: Shutdown hooks must be prefixed with rc.shutdown_pre or
```

Comment:
We could change the suffixes to `.boot` and `.shutdown` instead of `.sh` and nothing. I'm sure there's better names than my two examples but you get the idea.

Reply:
sounds good

Reply:
done